hazelcast/src/test/java/com/hazelcast/sql/impl/SqlTestSupport.java

```diff
@@ -116,77 +104,6 @@ public static MapContainer getMapContainer(IMap<?, ?> map) {
         return serialize(original);
     }
 
-    public static ListRowBatch createMonotonicBatch(int startValue, int size) {
```

Comment:
Do we still need this class? Should not all the tests extend Jet version now?

Reply:
It's planned to remove and enrich Jet version with necessary methods, yes.

Reply:
I'll do it in separate PR, too much changes.